virttest/test_setup.py

```diff
@@ -429,7 +429,10 @@ def __init__(self, params):
         self.hugepage_force_allocate = params.get("hugepage_force_allocate",
                                                   "no")
         self.suggest_mem = None
-        self.lowest_mem_per_vm = int(params.get("lowest_mem", "256"))
+        normalize_data_size = utils_misc.normalize_data_size
+        vm_mem_minimum = params.get("vm_mem_limit", "512M")
```

Comment:
`vm_mem_limit` is the max value,  and `vm_mem_minimum` is minimum value.

Reply:
@yanan-fu Yeah, my mistake. Updated.